<a href="https://colab.research.google.com/github/Sedowlas/EDA/blob/main/Cars_India_SQL.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

#Заливка данных

https://drive.google.com/file/d/1x1y_a1pYjZl-ICv1VZTd16sYQovWQ1ge/view?usp=sharing

In [ ]:
! gdown --id 1x1y_a1pYjZl-ICv1VZTd16sYQovWQ1ge

/usr/local/lib/python3.7/dist-packages/gdown/cli.py:131: FutureWarning: Option `--id` was deprecated in version 4.3.1 and will be removed in 5.0. You don't need to pass it anymore to use a file ID.
  category=FutureWarning,
Downloading...
From: https://drive.google.com/uc?id=1x1y_a1pYjZl-ICv1VZTd16sYQovWQ1ge
To: /content/newCarsIndia.csv
100% 22.2k/22.2k [00:00<00:00, 29.0MB/s]


#Предобработка

In [ ]:
!gdown --id 1BSHIKQ7rFw5BpTq5nw1UZfjPK_7Mpnbi
!mv _sqlite3.cpython-37m-x86_64-linux-gnu.so /usr/lib/python3.7/lib-dynload/
import os
os.kill(os.getpid(), 9)

/usr/local/lib/python3.7/dist-packages/gdown/cli.py:131: FutureWarning: Option `--id` was deprecated in version 4.3.1 and will be removed in 5.0. You don't need to pass it anymore to use a file ID.
  category=FutureWarning,
Downloading...
From: https://drive.google.com/uc?id=1BSHIKQ7rFw5BpTq5nw1UZfjPK_7Mpnbi
To: /content/_sqlite3.cpython-37m-x86_64-linux-gnu.so
100% 6.50M/6.50M [00:00<00:00, 145MB/s]


In [ ]:
import pandas as pd
import numpy as np

In [ ]:
import sqlite3

In [ ]:
con = sqlite3.connect('db')

In [ ]:
df = pd.read_csv('/content/newCarsIndia.csv')

In [ ]:
df.head()

Car              Style               Range  \
0           Tata Nexon EV        Compact SUV  312 Km/Full Charge   
1  Hyundai Grand i10 Nios  Compact Hatchback    20.2 - 26.2 Km/l   
2  Maruti Suzuki Alto 800    Entry Hatchback   24.7 - 31.39 Km/l   
3              Tata Punch        Compact SUV                 NaN   
4               MG Hector                SUV   13.96 - 17.4 Km/l   

       Transmission               VehicleType        PriceRange  \
0         Automatic                  Electric  ₹ 13.99 - 17.4 L   
1        AMT,Manual  Petrol,Petrol+CNG,Diesel    ₹ 5.3 - 8.51 L   
2            Manual         Petrol,Petrol+CNG   ₹ 3.39 - 5.03 L   
3        AMT,Manual                    Petrol   ₹ 5.83 - 9.49 L   
4  Automatic,Manual      Petrol,Diesel,Hybrid  ₹ 13.5 - 19.36 L   

  Rating(outof10)  
0             8.5  
1             8.5  
2             6.6  
3              na  
4             7.5

In [ ]:
df.columns = df.columns.str.lower()

In [ ]:
df.columns = df.columns = ['car', 'style', 'range', 'transmission', 'vehicletype', 'pricerange',
       'rating']

In [ ]:
df.isna().mean()

car             0.000000
style           0.000000
range           0.043137
transmission    0.000000
vehicletype     0.000000
pricerange      0.000000
rating          0.000000
dtype: float64

In [ ]:
df.to_sql('cars',con,index=False,if_exists='replace')

In [ ]:
def select(sql):
  return pd.read_sql(sql,con)

In [ ]:
pd.set_option('display.max_rows', None)

#Анализ

In [ ]:
sql = '''select * from cars t'''

In [ ]:
select(sql)

car                   style  \
0                         Tata Nexon EV             Compact SUV   
1                Hyundai Grand i10 Nios       Compact Hatchback   
2                Maruti Suzuki Alto 800         Entry Hatchback   
3                            Tata Punch             Compact SUV   
4                             MG Hector                     SUV   
5                            Tata Nexon          Subcompact SUV   
6                  Maruti Suzuki Baleno       Premium Hatchback   
7                         Hyundai Creta             Compact SUV   
8                  Maruti Suzuki Ertiga             Compact MPV   
9                            Kia Seltos             Compact SUV   
10                Maruti Suzuki Wagon R         Entry Hatchback   
11                        Mahindra Thar             Compact SUV   
12                     Mahindra Scorpio             Compact SUV   
13          Maruti Suzuki Vitara Brezza          Subcompact SUV   
14                        Toyota Glanza       Premium Hatchback   
15                      Mahindra XUV700                     SUV   
16                        Hyundai Venue          Subcompact SUV   
17                    Hyundai Elite i20       Premium Hatchback   
18                          Honda Amaze        Subcompact Sedan   
19               Maruti Suzuki S-Presso           Compact Sedan   
20                           Tata Tigor        Subcompact Sedan   
21                          Tata Altroz       Premium Hatchback   
22                          Tata Safari                     SUV   
23                      Hyundai Alcazar             Compact SUV   
24                        Hyundai Verna           Compact Sedan   
25                      Mahindra XUV300          Subcompact SUV   
26                 Toyota Urban Cruiser          Subcompact SUV   
27                           Honda City           Compact Sedan   
28                             MG Astor             Compact SUV   
29                           Tata Tiago         Entry Hatchback   
30                    Volkswagen Taigun             Compact SUV   
31                              Audi Q7    Premium Fullsize SUV   
32                        Tata Tigor EV        Subcompact Sedan   
33                       Nissan Magnite          Subcompact SUV   
34                       Tata Tiago NRG         Entry Hatchback   
35                         Skoda Kushaq              Luxury SUV   
36                  Maruti Suzuki Dzire        Subcompact Sedan   
37                         Toyota Hilux                     SUV   
38                      Mahindra Bolero                     SUV   
39                       Renault Triber             Compact MPV   
40                        Renault Kiger          Subcompact SUV   
41                 Toyota Innova Crysta    Fullsize/Premium MPV   
42                               BMW i8         Supersports Car   
43                         Renault Kwid         Entry Hatchback   
44               Mahindra Bolero Camper                     SUV   
45               Mahindra Bolero Pik-Up                     SUV   
46                           Bajaj Qute             Quadricycle   
47                      Toyota Vellfire    Fullsize/Premium MPV   
48                            Kia Sonet          Subcompact SUV   
49                Maruti Suzuki Celerio         Entry Hatchback   
50                Mercedes-Benz G-Class    Premium Fullsize SUV   
51                   Maruti Suzuki Eeco          Subcompact MPV   
52                    Tata Nexon EV Max             Compact SUV   
53                  Mahindra Bolero Neo          Subcompact SUV   
54                             MG ZS EV             Compact SUV   
55                         Jeep Compass             Compact SUV   
56                Mercedes-Benz C-Class   Premium Midsize Sedan   
57                    Maruti Suzuki XL6    Fullsize/Premium MPV   
58                    Mercedes-AMG G 63    Premium Fullsize SUV   
59              

In [ ]:
sql = '''select t.car,
count(1) as cnt,
avg(t.rating) as rating_mean


 from cars t
 group by t.car
 '''

In [ ]:
select(sql)

car  cnt  rating_mean
0                     Aston Martin DB11    1          0.0
1                      Aston Martin DBX    1          0.0
2                  Aston Martin Vantage    1          0.0
3                               Audi A4    1          7.5
4                               Audi A5    1          7.8
5                               Audi A6    1          7.6
6                               Audi A8    1          7.5
7                           Audi E-Tron    1          0.0
8                        Audi E-Tron GT    1          0.0
9              Audi E-Tron GT Sportback    1          0.0
10                              Audi Q2    1          0.0
11                              Audi Q5    1          7.5
12                              Audi Q7    1          8.4
13                              Audi Q8    1          8.5
14                              Audi R8    1          7.5
15                    Audi RS E-Tron GT    1          0.0
16                           Audi RS Q8    1          0.0
17                             Audi RS5    1          7.1
18                   Audi RS7 Sportback    1          8.1
19                              Audi S5    1          7.9
20                              Audi TT    1          8.3
21              BMW 2 Series Gran Coupe    1          0.0
22                         BMW 3 Series    1          8.2
23          BMW 3 Series Gran Limousine    1          0.0
24            BMW 3 Series Gran Turismo    1          7.8
25                         BMW 5 Series    1          8.0
26            BMW 6 Series Gran Turismo    1          8.0
27                         BMW 7 Series    1          7.7
28                         BMW 8 Series    1          8.5
29                               BMW M2    1          0.0
30                               BMW M3    1          8.6
31                               BMW M4    1          8.4
32                               BMW M5    1          7.6
33                               BMW M8    1          8.8
34                               BMW X1    1          8.0
35                               BMW X3    1          7.5
36                             BMW X3 M    1          0.0
37                               BMW X4    1          8.7
38                               BMW X5    1          8.1
39                             BMW X5 M    1          7.6
40                               BMW X6    1          7.8
41                             BMW X6 M    1          7.5
42                               BMW X7    1          9.0
43                               BMW Z4    1          7.6
44                               BMW i8    1          8.4
45                               BMW iX    1          0.0
46                               BYD E6    1          0.0
47                           Bajaj Qute    1          0.0
48                     Bentley Bentayga    1          7.7
49                  Bentley Continental    1          8.1
50                  Bentley Flying Spur    1          7.3
51                     Bentley Mulsanne    1          6.6
52                  Citroen C5 Aircross    1          0.0
53                            Datsun Go    1          6.4
54                       Datsun Go Plus    1          6.9
55                       Datsun Redi GO    1          7.6
56                      Ferrari 488 GTB    1          8.8
57                   Ferrari 488 Spider    1          7.9
58                Ferrari 812 Superfast    1          0.0
59                   Ferrari California    1          7.9
60                   Ferrari F8 Tributo    1          0.0
61                    Ferrari GTC4Lusso    1          7.9
62                    Ferrari Portofino    1          0.0
63                         Ferrari Roma    1          0.0
64                Ferrari SF90 Stradale    1          0.0
65                  Force Motors Gurkha    1          6.1
66                        Ford EcoSport    1          8.2
67                       Ford Endeavour    1          8.2
68                            Ford

Узнали 2 машины с высшим рейтингом - 9.0
1. BMW X7
2. Ford Figo

In [ ]:
sql = '''select t.style, 
 count(1) as cnt,
 avg(t.rating) as rating_mean

from cars t 
group by t.style
'''

In [ ]:
select(sql)

style  cnt  rating_mean
0        Compact Hatchback    6     8.166667
1              Compact MPV    3     8.000000
2              Compact SUV   19     4.805263
3    Compact SUV/Crossover    4     5.675000
4            Compact Sedan    7     7.942857
5          Entry Hatchback   10     6.380000
6           Fullsize Sedan    2     7.650000
7     Fullsize/Premium MPV    5     7.620000
8         Luxury Hatchback    7     4.285714
9               Luxury SUV   15     5.740000
10             Midsize SUV    1     0.000000
11           Midsize Sedan    2     8.250000
12       Performance Sedan    3     8.100000
13     Premium Compact SUV    3     7.466667
14   Premium Compact Sedan    2     7.400000
15           Premium Coupe   11     2.236364
16    Premium Fullsize SUV   19     6.621053
17  Premium Fullsize Sedan    7     7.957143
18       Premium Hatchback    8     7.150000
19    Premium Luxury Sedan   11     6.790909
20     Premium Midsize SUV    9     4.755556
21   Premium Midsize Sedan   13     4.615385
22    Premium Sports Coupe   15     5.766667
23    Premium Sports Sedan    7     4.357143
24             Quadricycle    1     0.000000
25                     SUV   27     5.211111
26              Sports Car   12     6.041667
27          Subcompact MPV    4     3.075000
28          Subcompact SUV   11     4.390909
29        Subcompact Sedan    5     6.300000
30         Supersports Car    6     5.400000

Узнали что рейтинг больше у Midsize Sedan - 8.25

In [ ]:
sql = '''select t.range, 
 count(1) as cnt,
 avg(t.rating) as rating_mean

from cars t 
group by t.range
'''

In [ ]:
select(sql)

range  cnt  rating_mean
0                  None   11     1.981818
1          10 - 12 Km/l    1     7.900000
2               10 Km/l    2     4.250000
3    10.01 - 14.22 Km/l    1     8.300000
4      10.1 - 13.5 Km/l    1     8.000000
5     10.3 - 11.91 Km/l    1     8.500000
6            10.31 Km/l    1     7.800000
7    10.54 - 13.38 Km/l    1     9.000000
8             10.6 Km/l    1     0.000000
9             10.7 Km/l    1     0.000000
10    10.75 - 15.1 Km/l    1     7.700000
11            10.8 Km/l    2     8.500000
12    10.85 - 15.3 Km/l    1     7.800000
13           10.98 Km/l    1     8.200000
14              11 Km/l    2     7.950000
15            11.1 Km/l    1     7.100000
16     11.2 - 12.4 Km/l    1     5.500000
17   11.24 - 13.38 Km/l    1     8.100000
18       11.3 - 18 Km/l    1     8.300000
19            11.3 Km/l    2     8.650000
20            11.4 Km/l    1     7.900000
21            11.6 Km/l    1     8.100000
22           11.76 Km/l    1     0.000000
23   11.86 - 39.53 Km/l    1     7.700000
24              12 Km/l    3     2.500000
25           12.01 Km/l    1     7.000000
26    12.06 - 14.2 Km/l    1     7.800000
27            12.1 Km/l    1     0.000000
28           12.35 Km/l    2     4.100000
29     12.4 - 13.9 Km/l    1     8.200000
30           12.65 Km/l    1     8.100000
31   12.75 - 16.35 Km/l    1     7.100000
32           12.75 Km/l    1     0.000000
33           12.78 Km/l    1     8.200000
34            12.8 Km/l    1     7.100000
35   12.82 - 14.24 Km/l    1     8.700000
36   12.82 - 14.37 Km/l    1     7.600000
37   12.95 - 16.38 Km/l    1     7.600000
38       13 - 15.2 Km/l    1     0.000000
39         13 - 17 Km/l    2     8.350000
40              13 Km/l    3     2.633333
41   13.05 - 13.06 Km/l    1     7.900000
42    13.06 - 16.5 Km/l    1     8.300000
43     13.1 - 19.3 Km/l    1     7.800000
44   13.17 - 16.55 Km/l    1     7.500000
45            13.2 Km/l    1     6.500000
46           13.39 Km/l    1     0.000000
47            13.5 Km/l    1     7.500000
48            13.7 Km/l    1     0.000000
49    13.8 - 13.84 Km/l    1     7.400000
50            13.8 Km/l    1     7.000000
51           13.86 Km/l    1     0.000000
52            13.9 Km/l    1     8.000000
53           13.95 Km/l    1     7.800000
54    13.96 - 17.4 Km/l    1     7.500000
55    14.1 - 16.97 Km/l    1     8.200000
56           14.11 Km/l    1     7.600000
57           14.12 Km/l    1     7.300000
58     14.2 - 20.4 Km/l    1     0.000000
59           14.23 Km/l    1     7.700000
60   14.28 - 18.65 Km/l    1     8.000000
61     14.3 - 14.4 Km/l    1     7.700000
62            14.3 Km/l    3     8.033333
63            14.4 Km/l    5     6.240000
64           14.41 Km/l    1     7.000000
65           14.49 Km/l    1     6.900000
66            14.5 Km/l    3     4.866667
67    14.59 - 14.6 Km/l    1     7.200000
68            14.6 Km/l    1     8.100000
69     14.7 - 21.7 Km/l    1     8.200000
70      14.82 - 19 Km/l    1     0.000000
71   14.82 - 19.62 Km/l    1     8.000000
72      15 - 15.37 Km/l    1     7.300000
73      15 - 15.68 Km/l    1     7.900000
74       15 - 17.9 Km/l    1     8.000000
75              15 Km/l    3     4.900000
76            15.1 Km/l    1     7.900000
77    15.3 - 19.62 Km/l    1     0.000000
78            15.4 Km/l    1     7.900000
79   15.56 - 22.48 Km/l    1     8.000000
80   15.78 - 17.88 Km/l    1     0.000000
81           15.81 Km/l    1     8.400000
82         16 - 21 Km/l    1     8.700000
83              16 Km/l    5     7.720000
84   16.09 - 18.19 Km/l    1     8.300000
85            16.1 Km/l    2     3.750000
86   16.13 - 20.37 Km/l    1     8.200000
87           16.14 Km/l    1     5.500000
88            16.2 Km/l    1     5.400000
89   16.24 - 18.97 Km/l    1     8.300000
90   16.44 - 18.47 Km/l    1     0.000000
91            16.6 Km/l    2     7.850000
92     16.8 - 21.4 Km/l    1     8.400000
93    16.9 - 16.94 Km/l    1     7.700000
94       16.9 - 23 Km/l    1

Узнали лучший диапазон расхода по рейтингу для новых машин - 9.0
1. 	10.54 - 13.38 Km/l	

In [ ]:
sql = '''select t.transmission, 
 count(1) as cnt,
 avg(t.rating) as rating_mean

from cars t 
group by t.transmission
'''

In [ ]:
select(sql)

transmission  cnt  rating_mean
0  AMT,Automatic,Manual    3     5.333333
1            AMT,Manual   19     6.926316
2             Automatic  169     5.678107
3      Automatic,Manual   45     5.495556
4                Manual   19     4.994737

Видим лучшую трансмиссию по данным из рейтинга AMT,Manual

In [ ]:
sql = '''select t.vehicletype, 
 count(1) as cnt,
 avg(t.rating) as rating_mean

from cars t 
group by t.vehicletype
'''

In [ ]:
select(sql)

vehicletype  cnt  rating_mean
0                     Diesel   28     4.967857
1              Diesel,Hybrid    1     7.000000
2              Diesel,Petrol   10     8.000000
3          Diesel,Petrol+CNG    1     0.000000
4       Diesel,Petrol,Hybrid    1     7.700000
5                   Electric   15     2.240000
6                     Hybrid    8     6.275000
7       Hybrid,Petrol,Diesel    1     7.700000
8                     Petrol  127     5.628346
9              Petrol,Diesel   46     6.310870
10      Petrol,Diesel,Hybrid    2     7.850000
11             Petrol,Hybrid    2     8.050000
12         Petrol,Petrol+CNG   10     6.550000
13  Petrol,Petrol+CNG,Diesel    2     8.150000
14         Petrol,Petrol+LPG    1     5.400000

Узнали лучший тип топлива Diesel,Petrol	с оценкой в рейтинге - 8.0

In [ ]:
sql = '''select t.pricerange, 
 count(1) as cnt,
 avg(t.rating) as rating_mean

from cars t 
group by t.pricerange
'''

In [ ]:
select(sql)

pricerange  cnt  rating_mean
0                 ₹ 1 Cr    3     5.266667
1       ₹ 1.01 - 1.19 Cr    1     0.000000
2       ₹ 1.04 - 1.38 Cr    1     8.500000
3              ₹ 1.05 Cr    1     7.800000
4       ₹ 1.06 - 1.12 Cr    1     0.000000
5              ₹ 1.06 Cr    1     0.000000
6              ₹ 1.09 Cr    1     7.100000
7              ₹ 1.11 Cr    1     7.400000
8       ₹ 1.15 - 1.93 Cr    1     7.700000
9              ₹ 1.16 Cr    1     0.000000
10      ₹ 1.18 - 1.79 Cr    1     9.000000
11      ₹ 1.19 - 1.92 Cr    1     8.000000
12              ₹ 1.3 Cr    1     8.600000
13      ₹ 1.32 - 1.98 Cr    1     8.100000
14             ₹ 1.33 Cr    1     6.900000
15      ₹ 1.42 - 1.54 Cr    1     7.300000
16      ₹ 1.42 - 1.76 Cr    1     7.700000
17             ₹ 1.44 Cr    1     8.400000
18      ₹ 1.45 - 2.43 Cr    1     6.600000
19              ₹ 1.5 Cr    2     4.100000
20      ₹ 1.53 - 2.07 Cr    1     7.500000
21             ₹ 1.57 Cr    1     7.600000
22             ₹ 1.58 Cr    1     7.500000
23       ₹ 1.6 - 1.69 Cr    1     8.000000
24             ₹ 1.62 Cr    1     8.500000
25      ₹ 1.63 - 2.51 Cr    1     7.600000
26             ₹ 1.74 Cr    1     7.600000
27             ₹ 1.77 Cr    1     8.200000
28              ₹ 1.8 Cr    2     0.000000
29      ₹ 1.82 - 1.99 Cr    1     8.100000
30             ₹ 1.82 Cr    1     7.500000
31      ₹ 1.91 - 2.22 Cr    1     7.900000
32        ₹ 10 - 14.79 L    1     8.300000
33     ₹ 10.19 - 18.45 L    1     8.700000
34     ₹ 10.44 - 18.18 L    1     8.400000
35     ₹ 10.69 - 17.79 L    1     0.000000
36     ₹ 11.29 - 14.55 L    1     7.700000
37     ₹ 11.29 - 19.49 L    1     0.000000
38      ₹ 11.29 - 19.5 L    1     8.200000
39       ₹ 11.4 - 18.6 L    1     0.000000
40     ₹ 11.99 - 13.49 L    1     8.300000
41     ₹ 12.42 - 14.57 L    1     8.300000
42     ₹ 12.49 - 13.64 L    1     0.000000
43     ₹ 12.77 - 17.62 L    1     7.300000
44     ₹ 13.18 - 24.58 L    1     0.000000
45     ₹ 13.35 - 19.23 L    1     8.100000
46      ₹ 13.5 - 19.36 L    1     7.500000
47     ₹ 13.53 - 16.03 L    1     0.000000
48             ₹ 13.59 L    1     6.100000
49      ₹ 13.99 - 17.4 L    1     8.500000
50     ₹ 14.65 - 21.95 L    1     7.900000
51     ₹ 15.25 - 23.46 L    1     5.500000
52     ₹ 16.34 - 20.15 L    1     0.000000
53     ₹ 16.98 - 24.49 L    1     7.000000
54     ₹ 17.18 - 25.68 L    1     7.700000
55     ₹ 17.74 - 19.24 L    1     0.000000
56     ₹ 17.86 - 21.13 L    1     8.100000
57     ₹ 18.04 - 30.97 L    1     8.200000
58                ₹ 2 Cr    1     0.000000
59             ₹ 2.05 Cr    1     0.000000
60             ₹ 2.08 Cr    1     7.600000
61       ₹ 2.1 - 2.16 Cr    1     0.000000
62             ₹ 2.12 Cr    1     7.500000
63             ₹ 2.17 Cr    1     0.000000
64             ₹ 2.23 Cr    1     8.800000
65             ₹ 2.24 Cr    1     8.100000
66      ₹ 2.25 - 2.51 Cr    1     8.000000
67      ₹ 2.27 - 2.48 Cr    1     8.200000
68             ₹ 2.28 Cr    1     7.300000
69      ₹ 2.32 - 3.41 Cr    1     7.700000
70             ₹ 2.33 Cr    1     6.300000
71             ₹ 2.43 Cr    1     0.000000
72        ₹ 2.5 - 3.2 Cr    1     0.000000
73              ₹ 2.6 Cr    1     0.000000
74             ₹ 2.62 Cr    1     8.400000
75       ₹ 2.64 - 2.84 L    1     0.000000
76             ₹ 2.72 Cr    1     7.500000
77             ₹ 2.95 Cr    1     0.000000
78             ₹ 21.35 L    1     8.700000
79     ₹ 21.99 - 25.88 L    1     8.200000
80       ₹ 22.2 - 23.5 L    1     5.500000
81     ₹ 22.69 - 27.47 L    1     7.600000
82     ₹ 23.79 - 23.98 L    1     8.700000
83     ₹ 26.29 - 29.29 L    1     8.400000
84     ₹ 28.88 - 31.88 L    1     8.200000
85             ₹ 29.15 L    1     0.000000
86     ₹ 29.98 - 37.28 L    1     0.000000
87     ₹ 29.99 - 34.99 L    1     8.000000
88                ₹ 3 Cr    1     8.600000
89        ₹ 3.2 - 3.4 Cr    1     7.300000
90       ₹ 3.22 - 4.1 Cr    1     8.500000
91        ₹ 3.3 - 4.4 Cr    1     8.100000
92       ₹ 3.39

Узнали топ диапазон лучших цен на новые машины исходя из рейтинга:  
С оценкой 9.0  
1. ₹ 1.18 - 1.79 Cr
2. ₹ 7.75 - 8.2 L   
С оценкой 8.5  
1. ₹ 1.04 - 1.38 Cr  
2. ₹ 1.62 Cr	  
3.  ₹ 13.99 - 17.4 L	
4. ₹ 3.22 - 4.1 Cr	
5. ₹ 5.3 - 8.51 L
6. ₹ 9.29 - 15.37 L	
